src/python/grpcio/grpc/extensions/client_config.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+
+package grpc.extensions;
```

Comment:
I suggest we remove the proto from this PR. The proto should be reviewed together with the actual extension implementation.